tests/Unit/Crypt/RSA/ModeTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2013 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+require_once 'Crypt/RSA.php' ;
+
+class Unit_Crypt_RSA_ModeTest extends PhpseclibTestCase
+{
+    public function testEncryptionModeNone()
+    {
+        $plaintext = 'a';
+
+        $rsa = new Crypt_RSA();
+
+        extract($rsa->createKey());
```

Comment:
Not a fan of this at all as it hides data flows. Assign the result to an array instead?